### DISCUSSION TITLE
Heat vs electricity emission allocation in CHP

### DISCUSSION THREAD

**Discussion by brunolajoie** [electricitymaps/electricitymaps-contrib#4770] [Q&A]
Dear electricitymaps,

Could you give us some more intuition on how did you split Heat vs Electricity emissions allocation in combined heat & power (CHP) plants in your latest [release](https://github.com/electricitymaps/electricitymaps-contrib/pull/4739)?

According to your [notebook](https://colab.research.google.com/drive/15UeUHyhcjVw8fGLc0Zx_HELSx20cw_-e#scrollTo=HBBwA5fx9icB), 
> For CHP power plants, total verified emissions include emissions from heat generation and from electricity generation. They need to be divided into electricity and heat emissions. Heat emissions can be calculated from annual allocations. According to the following [model](https://docs.google.com/spreadsheets/d/1y27Ro5T1FeGoV9Bf8pPF52Tw4U0MX08zpJbtk9Gz7b8/edit?usp=sharing), heat emissions are calculated as:
>
> heat_emissions = allocations / f(t)
>
> where f(t) is a function that depends on the [heat benchmark](https://climate.ec.europa.eu/system/files/2021-10/policy_ets_allowances_bm_curve_factsheets_en.pdf) set by the European Commission, and carbon leakage. The carbon leakage coefficient defines the proportion of energy provided to carbon emitting activities that could be moved outside of the EU boundaries. For example, steel production would be considered an activity that is sensitive to carbon leakage as steel production could be exported outside of EU boundaries if the carbon prices would increase too much. Residential heating is on the other end not sensitive to carbon leakage.
>
> We assume that carbon leakage is 40% for all production modes and will use the corresponding heat emission factor.

More specifically:
❓ What's the link and rational between carbon leakage and heat-emissions allocation?

**Comment by VIKTORVAV99** [comment]
Unfortunately, I can't answer these myself and the data team is quite busy but I'm sure @FelixDQ or @pierresegonne can shed some light on the topic as soon as they have the time.

And while the changes were merged to master it has not been deployed just yet, but it should be soon though.

**Comment by pierresegonne** [comment]
Hey @brunolajoie thanks for your question,

> What's the link and rational between carbon leakage and heat-emissions allocation?

The ETS registry includes all emissions for power plants (electricity + heat). To remove heat related emissions, we use the free allocations that are given to the power plant in the ETS scheme. These free allocations are computed using a formula which uses carbon leakage as an input. Carbon leakage determines whether the activity that benefits from the free allocations could be exported to somewhere else in the world not subject to carbon credits. Essentially it reduces how strict the carbon cap is on activities that would move elsewhere if the regulation was too strict (e.g. metallurgy - making this up I don't know if it's true).

[This dataset](https://zenodo.org/record/7316186) does the same but with a higher granularity, essentially computing the carbon leakage factor per power plant by doing a simple regression. We on the other hand took a simplifying assumption related to carbon leakage and decided to set it a fixed value for all plants that have CHP. That value was chosen roughly as the mean you find if you do a per power plant analysis.
